src/notation/internal/notationinteraction.cpp

```diff
@@ -3880,16 +3880,12 @@ void NotationInteraction::addTextToTopFrame(TextStyleType type)
 
 mu::Ret NotationInteraction::canAddTextToItem(TextStyleType type, const EngravingItem* item) const
 {
-    if (!item) {
-        return false;
-    }
-
     if (isVerticalBoxTextStyle(type)) {
-        return item->isVBox();
+        return item && item->isVBox();
```

Comment:
NB currently this can never happen because the only calling function already handles this case